libs/mixer/docs/reference/music/play-melody.md

```diff
@@ -0,0 +1,38 @@
+# play Melody
+
+Play a short melody of notes composed in a string.
+
+```sig
+music.playMelody("", 120);
+```
+
+The melody is short series of notes composed in a string. The melody is played at a rate set by the **tempo** value you give. The melody string contains a sequence of notes formatted like this:
+
+``"E B C5 A B G A F "``
+
+The melody is shown in the ``||music:play melody||`` block as note symbols which also appear in the Melody Editor.
+
+```block
+music.playMelody("E B C5 A B G A F ", 120);
+```
+
+The medlodies are most often created in the Melody Editor from the block so that valid notes are chosen and the correct melody length is set.
```

Comment:
```suggestion
The melodies are most often created in the Melody Editor from the block so that valid notes are chosen and the correct melody length is set.
```